src/osd/OSD.h

```diff
@@ -672,6 +673,16 @@ class OSDService {
     return agent_ops;
   }
 
+  void agent_inc_high_count() {
+    Mutex::Locker l(agent_lock);
+    flush_mode_high_count ++;
+  }
+
+  void agent_dec_high_count() {
+    Mutex::Locker l(agent_lock);
+    flush_mode_high_count --;
+  }
+
```

Comment:
I think this means that if any PG in the cluster needs more flushing then all flushing will go faster, as opposed to just the PGs in the most-full pool.  That's the best we can do currently, but we might also consider making the agent queue a priority queue?  Hrm.